extensions/ql-vscode/test/vscode-tests/no-workspace/remote-queries/gh-api/gh-actions-api-client.test.ts

```diff
@@ -95,7 +93,7 @@ describe("gh-actions-api-client real responses", () => {
       return;
```

Comment:
Branch here to check if this can go: https://github.com/github/vscode-codeql/pull/2000